### DISCUSSION TITLE
OSM Badge link is not working

### DISCUSSION THREAD

**Discussion by mrQuidome** [liodali/osm_flutter#310] [Q&A]
When I place the OSM contiburo badge I get a hyperlink, when clicked I got the following error (and nothing happens):

I/UrlLauncher(27559): component name for https://www.openstreetmap.org/copyright is null
I/UrlLauncher(27559): component name for https://flutter.dev is null

Not sure how this should work, maybe I have to configure something?

**Comment by liodali** [comment]
sorry for late response 
i will check that but you can make your own badge and replace the default

**Comment by mrQuidome** [comment]
No worries I was testing everything in detail I am on holiday the next 6
weeks. Testing is not finished.
